wiki/People/The_Team/Accounts_management_team/en.md

```diff
@@ -0,0 +1,31 @@
+# Accounts management team
+
+*For a team that moderates certain subforums, see: [Support Team](/wiki/People/The_Team/Support_Team)*
+
+**The accounts management team** is a group of people administering user accounts, as well as accepting silence and restriction appeals.
+
+## Point of contact
+
+The team can **only** be reached by sending a mail to [support@ppy.sh](mailto:support@ppy.sh) or [accounts@ppy.sh](mailto:accounts@ppy.sh). The identities of its members are kept secret in order to protect them from unwanted attention.
+
+## Role and tasks
+
+*Main article: [Help Centre](/wiki/Help_Centre)*
+
+*See also: [Account Restrictions](/wiki/Help_Centre/Account_Restrictions)*
+
+You should contact this team regarding account-related topics that are out of your control. Please remember to provide as much context on your issue as possible and send a email from the address associated with your osu! account.
+
+### [accounts@ppy.sh](mailto:accounts@ppy.sh)
+
+- [Appeals of restrictions placed on your account](/wiki/Help_Centre/Account_Restrictions), including silences and profile content removals.
+- Instances of misbehaviour that you [wish to admit](/wiki/Reporting_Bad_Behaviour/Handling_Foul_Play#what-can-i-do-if-i've-broken-the-rules?).
+- [Lose of access to the email](/wiki/Help_Centre#sign-in) tied to your osu! account, or when your account is stolen.
+- Username [reverts and minor spelling corrections](/wiki/Help_Centre#name-changes).
+- [Bot account registrations](/wiki/Bot_Account).
+
+### [support@ppy.sh](mailto:support@ppy.sh)
+
+- [Payment issues](/wiki/en/Help_Centre#supporter) for orders from [osu!store](https://osu.ppy.sh/store/listing).
+- Technical issues that couldn't or haven't been resolved in [the Help forum](https://osu.ppy.sh/community/forums/5), such as [connectivity issues](/wiki/Help_Centre#online-features).
+- Concerns about [a moderator's behaviour](/wiki/Reporting_Bad_Behaviour#can-i-report-a-moderator?).
```

Comment:
You could also add `privacy@ppy.sh` which handles account deletion and data requests (as required by GDPR etc.)

Reply:
good point.
also, I frequently see questions like "how long does it take for support to reply to my email", are there any specified deadlines after which a person should send a follow-up email ("one business week" or something of that sort)?

Reply:
Depends in incoming load, but if someone hasn't received a response within 3 days then something is wrong. Depends on the query though (restriction appeals are treated with much lower priority than general support requests, or account security issues).

Probably stating "within a week" is fine as a blanket for all cases.